lib/hammer_cli_foreman_tasks/task.rb

```diff
@@ -21,21 +34,46 @@ def execute
     end
 
     class ListCommand < HammerCLIForeman::ListCommand
+      extend WithoutNameOption
+      extend ActionField
+
       output do
         field :id, _('ID')
         field :action, _('Name')
```

Comment:
We shoudl call it `_('Action')` as we do in UI

Reply:
In the UI it's Name:
![image](https://user-images.githubusercontent.com/5609929/40983704-de7b60c8-68e8-11e8-859d-b8c2de635825.png)

Reply:
Oh, in the tasks table, it shows `action` for the same - I lean towards the `action` naming, that we should update as well in the UI.